system/services/rendering/ContentRendererService.cfc

```diff
@@ -133,19 +133,25 @@ component {
 	}
 
 	public string function renderEnum(
-		  required string objectName
-		, required string propertyName
-		, required string data
+		  required string data
+		,          string enum         = ""
+		,          string objectName   = ""
+		,          string propertyName = ""
 		,          any    context      = "default"
 		,          string recordId     = ""
-		,          string enumRenderer = $getPresideObjectService().getObjectAttribute( arguments.objectName, "enumRenderer", "enumLabel" )
+		,          string enumRenderer = "enumLabel"
 	) {
+		if ( !$helpers.isEmptyString( arguments.objectName ) ) {
+			arguments.enumRenderer = $getPresideObjectService().getObjectAttribute( arguments.objectName, "enumRenderer", "enumLabel" );
```

Comment:
Previously to this commit, you were able to specify an `enumRenderer` that was different to what was in the object property definition.

Perhaps the default enumRenderer arguments value should be `""`. Then use `getObjectAttribute()` when objectName supplied and just "enumLabel" otherwise?

Also, should it really be the _object_ attribute and not the _property_ attribute??

Reply:
Updated to default enumRenderer value to `""` and replaced `getObjectAttribute()` with `getObjectPropertyAttribute()`.